### PR TITLE
Artefact editor

### PR DESCRIPTION
**Pull request by zmbq**
This pull request contains the final fixes to version 0.5, prior to the implementation of the permissions.

There's a Read Only indication on editions, a simple breadcrumb in the navbar so you can return to the scroll if you are in one of its internal pages, and the fix to issue #366.

There was also an issue with the Visual Studio Code problem matcher - the output of the Vue CLI changed, and the filenames were not extracted properly (making it impossible to move from a problem in the problem pane to the actual file).

### CODE REVIEW

File: src/components/text/text-sign.vue [hunk 1]
```diff
@@ -37,32 +37,27 @@ export default class SignComponent extends Vue {
...
-            return false;
-        }
-        return true;
+        return si.character && !si.isReconstructed;
     }
 }
 </script>
 
 <style lang="scss" scoped>
+@import "../../assets./styles/_variables.scss";
```
Comment (Bronson-Brown-deVost) [inline src/components/text/text-sign.vue]: My computer yelled at me that this is invalid (this is on Ubuntu 18.04).  I changed to `@import "@/assets/styles/_variables.scss";` and that seemed to make everyone happy.

File: src/views/artefact-editor/roi-layer.vue [hunk 2]
```diff
@@ -39,10 +46,16 @@ export default class RoiLayer extends Vue {
 </script>
 
 <style lang="scss" scoped>
+@import '../../assets./styles/_variables.scss';
```
Comment (Bronson-Brown-deVost) [inline src/views/artefact-editor/roi-layer.vue]: My computer yelled at me that this is invalid (this is on Ubuntu 18.04).  I changed to `@import "@/assets/styles/_variables.scss";` and that seemed to make everyone happy.

File: src/views/artefact-editor/text-side.vue [hunk 3]
```diff
@@ -221,11 +243,10 @@ export default class TextSide extends Vue {
 </script>
 
 <style lang="scss" scoped>
+@import '../../assets./styles/_variables.scss';
```
Comment (Bronson-Brown-deVost) [inline src/views/artefact-editor/text-side.vue]: My computer yelled at me that this is invalid (this is on Ubuntu 18.04).  I changed to `@import "@/assets/styles/_variables.scss";` and that seemed to make everyone happy.

File: src/views/home/components/EditionCard.vue [hunk 4]
```diff
@@ -67,12 +81,12 @@ export default class EditionVersionCard extends Vue {
 </script>
 
 <style lang="scss" scoped>
-
+@import '../../../assets./styles/_variables.scss';
```
Comment (Bronson-Brown-deVost) [inline src/views/home/components/EditionCard.vue]: My computer yelled at me that this is invalid (this is on Ubuntu 18.04).  I changed to `@import "@/assets/styles/_variables.scss";` and that seemed to make everyone happy.

**Comment by zmbq** [comment]
Fixed the relative paths and linting errors. Sorry about the linting errors, it was because of that extra commit Shaindel did - which belonged in another branch.

**Pull request merged**